### PR TITLE
Bug 2078769: Added language translation on filter items

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
**Fixes**: 
https://issues.redhat.com/browse/OCPBUGSM-43639

**Analysis / Root cause**: 
Language translation was not added to the filter dropdown text.

**Solution Description**: 
Added language translation on filter items

**Screen shots / Gifs for design review**: 
-----SECRETS---------

Before----
https://bugzilla.redhat.com/attachment.cgi?id=1875011

After-----
image

------PIPELINES-----------

BEFORE----
https://bugzilla.redhat.com/attachment.cgi?id=1875012

AFTER-----
image

------PIPELINERUNS--------

BEFORE----
https://bugzilla.redhat.com/attachment.cgi?id=1875013

AFTER-----
image

-----Pods----
After------
Screenshot 2022-05-06 at 12 36 45 PM

**Unit test coverage report**: 
NA

**Test setup:**
Verify filter dropdown items by switching the language in secrets, pipelines and pipelineruns pages.

**Browser conformance**: 
- [x] Chrome
- [x] Firefox
- [x] Safari
- [ ] Edge

### CODE REVIEW

**Review by divyanshiGupta — CHANGES_REQUESTED** [review]

**Comment by divyanshiGupta** [comment]
@lokanandaprabhu e2e tests are failing. PTAL and fix them as needed.

**Comment by lokanandaprabhu** [comment]
> @lokanandaprabhu e2e tests are failing. PTAL and fix them as needed.

Handled the issue.

File: frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts [hunk 1]
```diff
@@ -84,11 +85,11 @@ export enum ListFilterId {
...
-  [ListFilterId.Running]: 'Running',
-  [ListFilterId.Failed]: 'Failed',
-  [ListFilterId.Succeeded]: 'Succeeded',
-  [ListFilterId.Cancelled]: 'Cancelled',
-  [ListFilterId.Other]: 'Other',
+  [ListFilterId.Running]: i18next.t('public~Running'),
+  [ListFilterId.Failed]: i18next.t('public~Failed'),
+  [ListFilterId.Succeeded]: i18next.t('public~Succeeded'),
+  [ListFilterId.Cancelled]: i18next.t('public~Cancelled'),
+  [ListFilterId.Other]: i18next.t('public~Other'),
```
Comment (vikram-raj) [inline frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts:92]: ns for the pipelines-plugin package is `pipelines-plugin`. 
```suggestion
  [ListFilterId.Running]: i18next.t('pipelines-plugin~Running'),
  [ListFilterId.Failed]: i18next.t('pipelines-plugin~Failed'),
  [ListFilterId.Succeeded]: i18next.t('pipelines-plugin~Succeeded'),
  [ListFilterId.Cancelled]: i18next.t('pipelines-plugin~Cancelled'),
  [ListFilterId.Other]: i18next.t('pipelines-plugin~Other'),
```

Comment (lokanandaprabhu) [reply frontend/packages/pipelines-plugin/src/utils/pipeline-utils.ts]: modified the changes and added translation for pods as well.

**Review by vikram-raj — COMMENTED** [review]
Verified it.

/lgtm

**Pull request merged**